packit_service/service/api/copr_builds.py

```diff
@@ -41,22 +41,50 @@ class CoprBuildsList(Resource):
     @ns.expect(pagination_arguments)
     @ns.response(HTTPStatus.PARTIAL_CONTENT, "Copr builds list follows")
     def get(self):
-        """ List all Copr builds. From 'copr-builds' hash, filled by service. """
-        # I know it's expensive to first convert whole dict to a list and then slice the list,
-        # but how else would you slice a dict, huh?
-        result = []
-        for k, cb in CoprBuild.db().get_all().items():
-            if k == LAST_PK:
-                # made-up keys, remove LAST_PK
-                continue
-            cb.pop("identifier", None)  # see PR#179
-            result.append(cb)
-
-        first, last = indices()
-        resp = make_response(dumps(result[first:last]), HTTPStatus.PARTIAL_CONTENT)
-        resp.headers["Content-Range"] = f"copr-builds {first + 1}-{last}/{len(result)}"
-        resp.headers["Content-Type"] = "application/json"
-        return resp
+        """ List all Copr builds. """
+
+        # Return relevant info thats concise
+        # Usecases like the packit-dashboard copr-builds table
+
+        with get_sa_session() as session:
+            builds_list = session.query(CoprBuild).all()
```

Comment:
could you please create a method on the CoprBuild class which would do what you need and not import get_sa_session here? I'd love to have all the database logic in models.py so we don't duplicate code